pkg/pipeline/transform/transform_network_direction.go

```diff
@@ -0,0 +1,38 @@
+package transform
+
+import (
+	"github.com/netobserv/flowlogs-pipeline/pkg/api"
+	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+)
+
+const (
+	ingress = 0
+	egress  = 1
+)
+
+func reinterpretDirection(output config.GenericMap, info *api.DirectionInfo) {
+	output[info.IfDirectionField] = output[info.FlowDirectionField]
```

Comment:
We should check that the field specified by `info.FlowDirectionField` actually exists.